tensorbay/client/segment.py

```diff
@@ -206,9 +207,18 @@ def _upload_file(self, data: FileMixin) -> None:
             )
 
             self._put_binary_file_to_azure(url, local_path, post_data)
+        elif backend_type == "fps":
```

Comment:
what is the meaning of `fps`?

Reply:
File Proxy Service

Reply:
`fps` is weird? can we get a better name?